app/react/Settings/components/specs/ToggleChildren.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { ToggleChildren } from '../ToggleChildren';
+
+describe('Toggle children', () => {
+  let component: ShallowWrapper<typeof ToggleChildren> = shallow(
+    <ToggleChildren toggled={false} />
+  );
+
+  it('should not display children if not toggled', () => {
+    expect(component.find('.toggle-children-children').text()).toBe('');
```

Comment:
i believe this test is wrong, the reason why text() returns empty string is because there is no children in the component being tested
```
  let component: ShallowWrapper<typeof ToggleChildren> = shallow(
    <ToggleChildren toggled={false} />
  );

```

Reply:
Done